src/sels_fe/src/components/quiz/Quiz/QuizWithQuestionsCreate.jsx

```diff
@@ -0,0 +1,305 @@
+import React, { useEffect, useState } from "react"
+import { useDispatch, useSelector } from "react-redux"
+import { useHistory } from "react-router"
+
+import {
+  postQuestion,
+  postQuiz,
+  updateQuestion,
+  updateQuiz,
+} from "../../../store/actions"
+
+import { Stepper, StepButton, Step, Box } from "@mui/material"
+import { IconButton, Button, Typography } from "@material-ui/core"
+
+import { Delete } from "@material-ui/icons"
+import { Divider, Stack } from "@mui/material"
+
+import QuizForm from "./components/QuizForm"
+import QuestionsForm from "./../Question/components/QuestionForm"
+import ChoiceForm from "./../Choice/components/ChoiceForm"
+import AnswersSelect from "../Answer/components/AnswersSelect"
+
+const steps = [
+  "Add Quiz Details",
+  "Add Questions",
+  "Add Choices",
+  "Select Answers",
+]
+let currentQuiz = null
+let currentQuestions = null
```

Comment:
Confirmation: What is the purpose of both variables, why are they not maintained with states?

Reply:
Will change both to states.